src/jdk.jartool/share/classes/sun/tools/jar/Main.java

```diff
@@ -802,18 +807,20 @@ private void expand(File dir, String[] files, Set<String> cpaths, int version)
                 Entry e = new Entry(f, name, false);
                 if (isModuleInfoEntry(name)) {
                     moduleInfos.putIfAbsent(name, Files.readAllBytes(f.toPath()));
-                    if (uflag)
+                    if (uflag) {
                         entryMap.put(name, e);
+                    }
```

Comment:
Thank you for cleaning up the if missing braces in this method.  Looks much better :-)